packages/request-client.js/src/api/request-network.ts

```diff
@@ -107,6 +187,31 @@ export default class RequestNetwork {
     return request;
   }
 
+  /**
+   * Persists an in-memory request to the data-access layer.
+   *
+   * @param transactionData The transaction data containing the request information
+   * @param requestId The ID of the request
+   * @param topics Optional topics for indexing the request
+   * @returns The result of the persist transaction operation
+   * @throws Error if the data access instance does not support persistence
+   */
+  public async persistRequest(
```

Comment:
When you use it in the tests:
```

      const persistResult = await newRequestNetwork.persistRequest(
        request.transactionData!,
        request.requestId,
        request.topics,
      );
```

Why splitting the needed data in 3 arguments when we expect the builder to store it all in 1 request object?

Reply:
Fixed. :white_check_mark: 
`persistRequest()` now takes the whole `Request` instead of 3 separate arguments.